content/response-to-reviewers.md

```diff
@@ -179,6 +185,13 @@ e.g. I see a lot of utility in manubot-style approaches for Carpentries-style on
 I assume this happens but want to double check (and am at the far end of a lousy Internet connection right now).
 So this is really more of an "@ctb todo" item.
 
+We discuss this comment further in [GH121](https://github.com/greenelab/meta-review/issues/121).
```

Comment:
Are we leaving #121 open until other aspects are addressed?

Reply:
I was planning to close it with this PR after opening a rootstock issue.